instance/models/mixins/database.py

```diff
@@ -221,6 +221,7 @@ def provision_mysql(self):
             _create_user(cursor, self.admin_user, self._get_mysql_pass(self.admin_user))
             _grant_privileges(cursor, "*", self.admin_user, "CREATE USER")
 
+            cursor.close()
```

Comment:
@viadanna Just to make sure, is this change still necessary after switching to S3 storage for integration tests?

Reply:
I don't think this is necessary for the current change, didn't really test without it. I left it as it seems a good practice.